test/jasmine/tests/funnelarea_test.js

```diff
@@ -1154,14 +1160,15 @@ describe('Test event data of interactions on a funnelarea plot:', function() {
             Plotly.restyle(gd, {
                 labels: [labels.concat(labels)],
                 values: [values.concat(values)]
-            });
-
-            click(pointPos[0], pointPos[1]);
-            expect(futureData.points.length).toEqual(1);
+            })
+            .then(function() {
+                click(pointPos[0], pointPos[1]);
+                expect(futureData.points.length).toEqual(1);
 
-            expect(futureData.points[0].pointNumber).toBeUndefined();
-            expect(futureData.points[0].i).toBeUndefined();
-            expect(futureData.points[0].pointNumbers).toEqual([0, 5]);
+                expect(futureData.points[0].pointNumber).toBeUndefined();
+                expect(futureData.points[0].i).toBeUndefined();
+                expect(futureData.points[0].pointNumbers).toEqual([0, 5]);
+            });
```

Comment:
`done`/`failTest`

Reply:
Good eye. Fixed in https://github.com/plotly/plotly.js/pull/5340/commits/2c0b913359024b733f76849a2e45f32842f2fd0e.